src/ru/geekbrains/HomeWork/Java2/Lesson3/Contact.java

```diff
@@ -0,0 +1,37 @@
+package ru.geekbrains.HomeWork.Java2.Lesson3;
+
+import java.util.*;
+
+public class Contact {
+
+    private int counts = 1;
+
+    public static Map<Integer, String> contact = new HashMap<>();
+
+    public static void see(){
+        for (Map.Entry<Integer, String> item : contact.entrySet()) {
+            System.out.printf("Key: %s  Value:%s  \n", item.getKey(),item.getValue());
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.